scene/gui/tree.cpp

```diff
@@ -1966,11 +1970,17 @@ void Tree::update_item_cell(TreeItem *p_item, int p_col) {
 			valtext = String::num(p_item->cells[p_col].val, Math::range_step_decimals(p_item->cells[p_col].step));
 		}
 	} else {
-		valtext = p_item->cells[p_col].text;
+		// Don't auto translate if it's in string mode and editable, as the text can be changed to anything by the player.
```

Comment:
```suggestion
		// Don't auto translate if it's in string mode and editable, as the text can be changed to anything by the user.
```
I think it's more accurate in GUI context.